tutor/plugins/openedx.py

```diff
@@ -142,3 +142,63 @@ def is_directory_mounted(image_name: str, dirname: str) -> bool:
 hooks.Filters.ENV_TEMPLATE_VARIABLES.add_item(
     ("iter_mounted_directories", iter_mounted_directories)
 )
+
+
+@hooks.Filters.LMS_WORKER_COMMAND.add(priority=hooks.priorities.HIGH)
+def _add_default_lms_worker_parameters(
+    command: list[str]
+) -> list[str]:
+    command = [
+        "celery",
+        "worker",
+        "--app=lms.celery",
+        "--loglevel=info",
+        "--hostname=edx.lms.core.default.%%h",
+        "--queues=edx.lms.core.default,edx.lms.core.high,edx.lms.core.high_mem",
+        "--max-tasks-per-child=100",
+    ]
+    return command
+
+
+@hooks.Filters.CMS_WORKER_COMMAND.add(priority=hooks.priorities.HIGH)
+def _add_default_cms_worker_parameters(
+    command: list[str]
+) -> list[str]:
+    command = [
+        "celery",
+        "worker",
+        "--app=cms.celery",
+        "--loglevel=info",
+        "--hostname=edx.cms.core.default.%%h",
+        "--queues=edx.cms.core.default,edx.cms.core.high,edx.cms.core.low",
+        "--max-tasks-per-child=100",
+    ]
+    return command
+
+
+@hooks.lru_cache
+def iter_cms_celery_parameters() -> t.Iterator[str]:
+    """
```

Comment:
These docstrings are useless because they duplicate the mypy signature, let's remove them.